Pirat/Program.cs

```diff
@@ -13,7 +13,13 @@ public class Program
         public static void Main(string[] args)
         {
             CheckEnvironmentVariables();
-            CheckConnectionToDatabase();
+            
+            DatabaseInitialization.CheckConnectionToDatabase();
+            var initTables = Environment.GetEnvironmentVariable("PIRAT_INIT_DB_TABLES_IF_NOT_EXIST")?.Equals("true", StringComparison.Ordinal);
+            var initData = Environment.GetEnvironmentVariable("PIRAT_INIT_DUMMY_DATA_IF_NOT_EXIST")?.Equals("true", StringComparison.Ordinal);
+            if (initTables.HasValue) DatabaseInitialization.InitDatabaseTables();
+            if (initData.HasValue) DatabaseInitialization.InitDatabaseWithDummyData();
```

Comment:
`HasValue` would also be true if the value is not equals `true`. It would be better to do the comparison with `Equals(String, String, StringComparison)` instead so that no null-check would be needed.

Reply:
You are right. I fix this